### ISSUE TITLE
Track test set performance during training

### ISSUE THREAD

**Issue by davidbenjamin** [davidbenjamin/permutect#19]
While it's cheating to use it to tune hyperparameters, such as early stopping, it would be very useful to track, say, AUROC for both the test and validation sets for each epoch of training and compare them.  This is especially so do to the mediocrity of our training and validation labels.
